CODEOWNERS

```diff
@@ -370,6 +370,9 @@ extensions/filters/http/oauth2 @derekargueta @mattklein123
 # Lua cluster specifier
 /*/extensions/router/cluster_specifiers/lua @StarryVae @wbpcode
 
+# SHA-1
+/*/extensions/hash/sha1 @zhaohuabing @wbpcode
```

Comment:
@zhaohuabing @wbpcode I made you the codeowners of this, because this is only used by basic_auth filter, which is owned by you. Is it ok?